src/main/java/org/gridsuite/explore/server/RestResponseEntityExceptionHandler.java

```diff
@@ -30,18 +29,16 @@ protected ResponseEntity<Object> handleException(RuntimeException exception) {
         }
         ExploreException exploreException = (ExploreException) exception;
         switch (exploreException.getType()) {
-            case STUDY_NOT_FOUND:
-                return ResponseEntity.status(HttpStatus.NOT_FOUND).body(STUDY_NOT_FOUND);
             case FILTER_NOT_FOUND:
-                return ResponseEntity.status(HttpStatus.NOT_FOUND).body(FILTER_NOT_FOUND);
-            case CONTINGENCY_LIST_NOT_FOUND:
-                return ResponseEntity.status(HttpStatus.NOT_FOUND).body(CONTINGENCY_LIST_NOT_FOUND);
+                return ResponseEntity.status(HttpStatus.NOT_FOUND).body(exploreException.getType());
             case NOT_ALLOWED:
                 return ResponseEntity.status(HttpStatus.FORBIDDEN).body(NOT_ALLOWED);
             case REMOTE_ERROR:
                 return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(exception.getMessage());
             case UNKNOWN_ELEMENT_TYPE:
-                return ResponseEntity.status(HttpStatus.INTERNAL_SERVER_ERROR).body(UNKNOWN_ELEMENT_TYPE);
+            case INSERT_STUDY_FAILED:
+            case NOTIFICATION_DIRECTORY_CHANGED:
+                return ResponseEntity.status(HttpStatus.INTERNAL_SERVER_ERROR).body(exploreException.getType());
```

Comment:
why adding exception type in the body for some exception types, and not for other ?

Reply:
i will keep the same file i will only remove STUDY_NOT_FOUND, CONTINGENCY_LIST_NOT_FOUND because they are not used